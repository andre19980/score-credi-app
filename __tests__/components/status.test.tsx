import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import Status from "@/components/status";

describe("Status", () => {
  it("approved case", () => {
    const { container } = render(<Status status="true;10000" />);
    expect(container).toMatchSnapshot();
  });

  it("denied case", () => {
    const { container } = render(<Status status="false" />);
    expect(container).toMatchSnapshot();
  });
});
