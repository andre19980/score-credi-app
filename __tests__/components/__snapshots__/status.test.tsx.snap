// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status approved case 1`] = `
<div>
  <div
    class="flex flex-col h-full justify-center sm:col-start-2 sm:col-span-2"
  >
    <div
      class="flex flex-col bg-gray-50 p-8 items-center gap-y-6 rounded"
    >
      <img
        alt="status image"
        data-nimg="1"
        decoding="async"
        height="180"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
        style="color: transparent;"
        width="180"
      />
      <p
        class="text-gray-900 sm:text-sm sm:leading-6"
      >
        Parabéns! Seu crédito de R$ 10.000 foi aprovado e estará disponível em breve.
      </p>
    </div>
  </div>
</div>
`;

exports[`Status denied case 1`] = `
<div>
  <div
    class="flex flex-col h-full justify-center sm:col-start-2 sm:col-span-2"
  >
    <div
      class="flex flex-col bg-gray-50 p-8 items-center gap-y-6 rounded"
    >
      <img
        alt="status image"
        data-nimg="1"
        decoding="async"
        height="180"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
        style="color: transparent;"
        width="180"
      />
      <p
        class="text-gray-900 sm:text-sm sm:leading-6"
      >
        Infelizmente você não preencheu os requisitos para a aprovação do crédtio. Para mais informações, consulte nosso material no site oficial.
      </p>
    </div>
  </div>
</div>
`;
